chains/evm/cli/account/generate-key-pair.go

```diff
@@ -11,6 +12,10 @@ var generateKeyPairCmd = &cobra.Command{
 	Short: "Generate bridge keystore (Secp256k1)",
 	Long:  "The generate subcommand is used to generate the bridge keystore. If no options are specified, a Secp256k1 key will be made.",
 	RunE:  generateKeyPair,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		confirmTransfer(cmd, args)
```

Comment:
Why we need to confirm this command explicitly?

Reply:
Everything else looks great

Reply:
This shouldn't be here, seems like copy-paste mistake :sweat: